Server/docker-compose.yml

```diff
@@ -51,3 +53,38 @@ services:
       - 8443:8443
       - 8444:8444
       - 9229:9229
+
+  directus:
+    build:
+      context: ../../citrineos-core/DirectusExtensions
+      dockerfile: ./directus.Dockerfile
+    ports:
+      - 8055:8055
+    volumes:
+      - ../../citrineos-core/Server/data/directus/uploads:/directus/uploads
```

Comment:
wouldnt this just be `./Server/data` instead of `../../citrineos-core` ? I think since someone may just cloned ocpi directly, maybe they wouldnt have citrineos-core potentially?